### ISSUE TITLE
Feature Request: Streaming files

### ISSUE THREAD

**Issue by GameOverFlowChart** [oxen-io/session-android#1053]
So the max file size in session is very limited and it's understandable why that is how it is. I know that increasing the limit as a premium feature is planned. This request is something different. Session has the function to stream video, it's in Beta right now and it's p2p but in future it will be onion rooted. It should be possible (and my request is it to make it possible) to use similar technic to stream files to contacts without file size limitations. The files wouldn't be saved on file servers. I expect the speed to be the same as than video is streamed in real time. The limitation would be that both sender and reciver would need to be online at the same time. If one side loses connection the stream would pause atomatically and start again as soon as both are online. This limitation would also mean that there will still be a reason to opt in for the pro feature so this could become a free alternative to send files of any size. Streaming a file should always need the other end to accept the request like it is the case with a video call. Accepting a streamed file would mean that both users share their online status with each other until the stream is over so that the automatic pause and resume would work. The load on the network shouldn't be more than the load during a video call, so if live video is possible, streaming files should also be possible.

**Comment by 0x330a** [comment]
In future when lokinet is integrated as a library you will be able to open a stream across devices and preserve anonymity. This will enable large file transfers where the only limits are the devices being online and is on the roadmap

**Comment by GameOverFlowChart** [comment]
Yes I knew that it would be possible with lokinet, it's great to hear that the feature is on the roadmap for session.

**Comment by GameOverFlowChart** [comment]
> In future when lokinet is integrated as a library you will be able to open a stream across devices and preserve anonymity. This will enable large file transfers where the only limits are the devices being online and is on the roadmap

I realize that I misunderstood your reply. You said that large file transfers will be enabled by Lokinet (this is something I already knew as I made this request) and I interpreted it as if my requested feature is already on the roadmap. But I can't see this on the new roadmap here: https://oxen.io/roadmap

I would expect it to be on the tree similar to "Onion-Routed Calls".
